include/glow/Optimizer/GraphOptimizer/CompilationContext.h

```diff
@@ -116,6 +116,9 @@ struct OptimizationOptions {
   /// The number of cores to assign to non-SLS partition when using SparseNN
   /// partitioning scheme
   unsigned int sparseNNPartitioningSchemeNumCoresOther{1};
+
+  /// If true does int64 to int32 type demotion
```

Comment:
Can you expand here to make clear it will only do the conversion if the backend also supports it.